start.go

```diff
@@ -44,10 +38,10 @@ func start(c *cli.Context) error {
 		"--name", "fnserver",
 		"-v", fmt.Sprintf("%s/data:/app/data", wd),
 		"-v", "/var/run/docker.sock:/var/run/docker.sock",
-		"-p", "8080:8080",
```

Comment:
Why change this?

Reply:
because it can be really whatever it can, you have FN_PORT which would change port, so it's not always 8080.

Reply:
I'd much prefer this: https://github.com/fnproject/cli/pull/79 vs what you have here.

Reply:
okay, will revert.